src/bento.js

```diff
@@ -3,9 +3,13 @@
  * Entry point for `bento.js`
  */
 
+import {isEsm} from '#core/mode';
+
 import {install as installCustomElements} from './polyfills/custom-elements';
 
-installCustomElements(self);
+if (!isEsm()) {
+  installCustomElements(self);
+}
```

Comment:
This is still needed for Firefox 60. If we want to support native CEv1 in ESM:

```suggestion
installCustomElements(self, class {});
```

This will install:
- a polyfill in browsers with CEv1 (including Firefox 60)
- a minimal wrapper in CEv1 browsers that load nomodule JS
- nothing in CEv1 browsers that load ESM

Reply:
Gotcha. If I understand correctly, if you were to set `async` on the `bento.mjs` tag, you can no longer support polyfillable browsers that load `type=module`?

Reply:
`type=module` implies `deferred`, which is very similar to `async`. So there's a race condition that ESM browsers that need a full polyfill will try to access `window.customElements` before it can be installed.

We could chose to not support Firefox 60-63 (inclusive). In which case, we should still change this to:

```suggestion
if (!isEsm()) {
  installCustomElements(self, class {});
}
```

That way browsers that have CEv1 (but are loading the `nomodule` build for whatever reason) can use the native implementation with minimal wrapper.

Reply:
Thanks for clarifying. Going with your last approach.

I've updated the documentation to show the preferred module/nomodule syntax. Let me know if that makes sense to you.